Mathlib/Tactic/Linarith/Oracle/FourierMotzkin.lean

```diff
@@ -304,9 +304,12 @@ from the `linarith` state.
 -/
 def elimVarM (a : ℕ) : LinarithM Unit := do
   let vs ← getMaxVar
-  if (a ≤ vs) then (do
+  if (a ≤ vs) then
+    Lean.Core.checkSystem "Linarith.elimVarM"
```

Comment:
```suggestion
  Lean.Core.checkSystem decl_name%.toString
```

Reply:
I thought about this, but I think this makes the check more expensive as every call then has to convert from a name to a string before performing the check. As fast as I can tell the conversion isn't inlineanle since it involves a memory allocation.

Reply:
The conversion will be done only once, because `decl_name%.toString` is a closed term.

Reply:
Done.